.circleci/config.yml

```
@@ -2063,9 +2068,11 @@ workflows:
           context:
             - slack
 
-  scheduled-kontrol-tests:
+  develop-kontrol-tests:
     when:
-      equal: [ build_four_hours, <<pipeline.schedule.name>> ]
+      or:
+        - equal: [ "develop", <<pipeline.git.branch>> ]
+        - equal: [ true, <<pipeline.parameters.kontrol_dispatch>> ]
```

This will wind up running for every scheduled trigger as well.  Not sure I'll get the indenting right here, but it should be like the `develop-fault-proofs` job above.
```suggestion
      and:
        - or:
          - equal: [ "develop", <<pipeline.git.branch>> ]
          - equal: [ true, <<pipeline.parameters.kontrol_dispatch>> ]
        - not:
            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
```

---

Ah thank you, fixed in d702f2072